### DISCUSSION TITLE
The page source appears blank

### DISCUSSION THREAD

**Discussion by dima-online** [requarks/wiki#3287] [Error / Bug Report]
If the page is large or there are many embedded images on the page (img src = "data:image/png; base64), then the source code of the page is not displayed. I suspect the problem is somewhere in the javascript that displays the page.

The source texts for Wiki were prepared in Winword. Pictures in Winword were inserted from different copy-paste sources. 
After that, the text with pictures was inserted into the wiki by copy-paste. This is how embedded images appeared (img src = "data:image/png; base64).

Wiki.js 2.5.170
 - OS: Docker
 - Wiki.js version: 2.5.170
 - Database engine: docker postgres:11

I am attaching an example of such a page (sorry - the page is in Russian)
[VDC-template.zip](https://github.com/Requarks/wiki/files/5735215/VDC-template.zip)

**Comment by ghost** [comment]
You haven't provided the required info about your host! (OS, Wiki.js version, Database engine)
